### DISCUSSION TITLE
8.16.0 support windows 7 ? need install other dependencies?

### DISCUSSION THREAD

**Discussion by ovelin** [libvips/libvips#4328] [General]
8.16.0 support windows 7 ? need install other dependencies?

**Comment by kleisauke** [comment]
The v8.16.0 Windows binaries [ought to work with Windows 7](https://github.com/libvips/build-win64-mxe/blob/v8.16.0/build/plugins/llvm-mingw/llvm-mingw.mk#L14-L17). However, the toolchain used to build the binaries uses UCRT (the Universal C Runtime) which is only preinstalled since Windows 10, but can be [installed](https://www.microsoft.com/en-us/download/details.aspx?id=50410) on Vista or newer.

Note that all versions older than Windows 10 are end-of-life (EOL). It's possible that the binaries may not work due to dependencies updating their minimum supported Windows version.

**Reply by ovelin** [reply]
ok，thanks
